addon/components/polaris-action-list.js

```diff
@@ -53,17 +52,19 @@ export default Component.extend({
    * @type {function}
    * @default no-op
    */
-  onActionAnyItem() {},
+  onActionAnyItem() {}
 
   /**
    * @private
    */
-  hasMultipleSections: gt('finalSections.length', 1).readOnly(),
+  @(gt('finalSections.length', 1).readOnly())
```

Comment:
wouldn't this be,
```javascript
@readOnly
@gt('finalSections.length', 1)
hasMultipleSections
```

Reply:
ah yeah looks like the codemod also didn't set all the `@readOnly` decorators properly 😞

Reply:
update, looks like the codemod is doing the right thing. If trying to use `@readOnly` on its own, it errors with

```
You attempted to use @readOnly as a decorator directly, but it requires a `dependentKey` parameter
```

If I pass any keys into it, it errors with

```
Only one computed property decorator can be applied to a class field or accessor, but 'hasMultipleSections' was decorated twice.
```

🤷‍♂

Reply:
> If trying to use @readOnly on its own, it errors

I guess it's trying to use the `readOnly` CP macro, instead of the modifier (same as the old `import { readOnly } from '@ember/object/computed'` 🤔 

In ES6-land, maybe these are defined with a getter and no setter, making them read-only anyway? Not sure...

Reply:
Yeah seems like it, and I don't see any `readOnly` available as an import from `ember-decorators` currently so I guess for now we have to stick with `.readOnly()`

Reply:
Not much else we can do at this point by the sounds of it then 🤷‍♂

Reply:
yeah, I think Tom & the codemod are correct 😄 

PS: same old `import { readOnly } from '@ember/object/computed'` should work now as a decorator for ` readOnly` CP, but this is a `readOnly` CP modifier (or whatever they are called)